app/src/components/LogoutSpotifyButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactElement } from "react";
+import { button } from "./ConnectSpotifyButton.css";
```

Comment:
`ConnectSpotifyButton.css` will not be necessary if we use the `Button` component.